src/ray/core_worker/store_provider/local_plasma_provider.h

```diff
@@ -18,9 +18,17 @@ class CoreWorkerLocalPlasmaStoreProvider : public CoreWorkerStoreProvider {
  public:
   CoreWorkerLocalPlasmaStoreProvider(const std::string &store_socket);
 
-  /// See `CoreWorkerStoreProvider::Put` for semantics.
   Status Put(const RayObject &object, const ObjectID &object_id) override;
 
+  /// See `CoreWorkerStoreProvider::Put` for semantics.
+  Status Create(const std::shared_ptr<Buffer> metadata,
```

Comment:
`const std::shared_ptr<Buffer>& metadata` will avoid a shared pointer increment when this function is called